src/config.py

```diff
@@ -14,5 +18,15 @@ def update_config():
   with open('config/2019.json', 'r') as config_file:
     config_json['2019'] = json.load(config_file)
 
+    for contributor in config_json['2019']['contributors'].items():
+      if 'gravatar' not in contributor[1] and 'teams' in contributor[1]:
+        contributor[1]['avatar_url'] = default_avatar_folder_path + contributor[1]['teams'][0] + "_avatar.jpg"
+      elif 'teams' not in contributor[1]:
+        contributor[1]['avatar_url'] = default_avatar_path
+      else:
+        gravatar_url = "https://www.gravatar.com/avatar/" + hashlib.md5(contributor[1]['gravatar'].lower().encode()).hexdigest() + ".jpg?"
```

Comment:
Can you also add `www.gravatar.com` to our [img-src CSP policy](https://github.com/HTTPArchive/almanac.httparchive.org/blob/f3883291d56f6af359aefa8c60b5e0b86136a53d/src/csp.py#L20)?